### ISSUE TITLE
Does your Dataset class cope with empty frames? 

### ISSUE THREAD

**Issue by GeorgePearse** [sovit-123/fasterrcnn-pytorch-training-pipeline#55]
I often use this repo as a point of reference. 

``` 
boxes = [] 
boxes = torch.as_tensor(boxes, dtype=torch.float32)
```
I don't believe the above would create the required shape to be provided to Faster RCNN in the case of an empty frame. See https://stackoverflow.com/questions/66063046/how-to-train-faster-rcnn-on-dataset-including-negative-data-in-pytorch . 

Does your dataset / model cope with empty frames in some other way? Just trying to find what's considered sensible practice.

**Comment by sovit-123** [comment]
@GeorgePearse 
Yes, it handles empty frames. This part in [datasets.py](https://github.com/sovit-123/fasterrcnn-pytorch-training-pipeline/blob/e0ed81307014182d47dd032a2f80a6c49a5fd18c/datasets.py#L333) handles it.
